### ISSUE TITLE
Introduzione sistema di gestione per gli utenti connessi

### ISSUE THREAD

**Issue by Dasc3er** [devcode-it/openstamanager#797]
## Comportamento richiesto

Al fine di garantire un maggiore controllo sugli utenti connessi al gestionale, può essere utile introdurre un sistema più complesso che permette di visualizzare gli utenti attivi e le relative attività per gli amministratori. Ciò può essere utile anche per migliorare la sicurezza.
Questo sistema sarebbe la base per la futura introduzione di un sistema di chat interno al gestionale, parallelo all'attuale procedura delle Note interne.

**Comment by Dasc3er** [comment]
Un'applicazione di questo sistema può essere vista nella gestione degli accessi nel caso di aggiornamenti e ripristini backup. La relativa procedura dovrebbe infatti provocare il logout immediato di tutte le utenze per impedire possibili conflitti, preferibilmente attraverso delle chiamate AJAX dedicate.

**Comment by lucasalva87** [comment]
Esatto, interessante prevedere messaggi in broadcast, dedicati al singolo utente o anche gruppi di utenti (Tecnici, Amministratori, Agenti ecc..). Da valutare eventuali sistemi di messaggistica già sviluppati open-source.
